apteryx.c

```diff
@@ -275,39 +331,23 @@ handle_watch (rpc_message msg)
         return true;
     }
 
+    root = break_up_trunk(rpc_msg_decode_tree (msg));
```

Comment:
do we need error checking on the return from rpc_msg_decode_tree ... all other cases seem to manage check for null return

Reply:
Done